components/camel-test-spring-junit5/pom.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Licensed to the Apache Software Foundation (ASF) under one or more contributor 
```

Comment:
License header is formatted as the others we have in pom files

Reply:
Indeed, the formatter looks not to be applied in pom files. I'll revert based on camel-test-spring.